.github/workflows/run-connector-tests-command.yml

```
@@ -19,13 +19,15 @@ on:
         description: "The pull request number, if applicable"
         required: false
         type: number
+  push:
 
 jobs:
   call-connector-ci-tests:
+    if: ${{ github.event_name != 'push' || (github.repository != 'airbytehq/airbyte' && (secrets.GCP_PROJECT_ID != '' || vars.GCP_PROJECT_ID != '')) }}
```

I believe this would be more readable:

```suggestion
    # Run always for 'workflow_dipatch' events.
    # Run for 'push' events on forks, but only if the fork defines "BYO Secrets".
    if: >
      github.event_name == 'workflow_dispatch' ||
      (github.event_name == 'push' && github.repository != 'airbytehq/airbyte' &&
      (secrets.GCP_PROJECT_ID != '' || vars.GCP_PROJECT_ID != ''))
```

I will auto-apply it.